docusaurus/docs/iOS/basics/logs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Logs
+---
+
+Using the Stream Chat SDK is straightforward, and you should have something running very quickly. First, however, you're likely to want to view the logs provided by the SDK. We surface most errors to the console from information, warnings and errors. This also includes logs provided by the Stream Chat API.
+
+**By default, logs are disabled.**
+
+![Screenshot shows Xcode with the customized logs in the console](../assets/log-messages.png)
+
+## Enable Logging
+
+You can enable logs by setting the `logLevel`.
+
+```swift
+import StreamChat
+
+LogConfig.level = .info
+```
+
+We have four different `logLevel`'s available.
+
+- `.info` (This will provide all the information logs to the console).
+- `.debug` (This is unfiltered and will show **ALL** logs).
```

Comment:
Why don't we move this up, and switch `error` and `warning` so levels are ordered by verbosity?

Reply:
Nice idea @b-onc I shall make this change 💯